package/osx/cmake/codesign-package.cmake

```diff
@@ -45,20 +43,15 @@ else()
    list(APPEND CODESIGN_FLAGS -s -)
 endif()
 
-list(APPEND CODESIGN_TARGETS "${CMAKE_INSTALL_PREFIX}/RStudio.app")
```

Comment:
I should gate-keep these changes to Electron only, just to avoid potentially breaking the existing desktop builds.